src/Adapter/Image/ImageRetriever.php

```diff
@@ -137,10 +137,16 @@ public function getProductImages(array $product, Language $language)
      */
     public function getImage($object, $id_image)
     {
+        static $psImageoptLazyResize = null;
+
         if (!$id_image) {
             return null;
         }
 
+        if ($psImageoptLazyResize === null) {
+            $psImageoptLazyResize = \Configuration::get('PS_IMAGEOPT_LAZY_RESIZE');
```

Comment:
I don't think we need to cache a boolean value like this 😄  . If some cache must happens, it should be in `Configuration::get` not in ImageRetriever::getImage